crates/lang/ir/src/ir/attrs.rs

```diff
@@ -511,6 +531,22 @@ impl core::fmt::Display for AttributeArg {
     }
 }
 
+/// Either a wildcard selector or a specified selector.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum SelectorOrWildcard {
+    Wildcard,
+    Selector(ir::Selector),
+}
+
+impl core::fmt::Display for SelectorOrWildcard {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> Result<(), core::fmt::Error> {
+        match self {
+            Self::Selector(selector) => write!(f, "{:?}", selector.to_bytes()),
```

Comment:
If `ir::Selector` already implements `Display` we should ideally just forward to its `Display` implementation using `core::fmt::Display::fmt(&selector, f)`.